.semgrep/adapter/bid-type-if-check.yml

```diff
@@ -0,0 +1,21 @@
+rules:
+  - id: bid-type-if-check
+    message: Prebid server expects media type set in adapter response. we recommend implementing a pattern where adapter server sets the [MType](https://github.com/prebid/openrtb/blob/53ac1b9ca7ddf860988ff69bf5018b32ced1fbc5/openrtb2/bid.go#L334) field in the response to accurately determine the media type for the impression.
+    languages:
+      - go
+    severity: WARNING
+    patterns:
+      - pattern-inside: |
+          if $CONDITION != nil {
+            return $ORTBTYPE, nil
```

Comment:
we can use `...` as second return params.  This will also cover functions that only returns `$ORTBTYPE`  and has no second param to return

Reply:
updated pattern in [28cadac](https://github.com/prebid/prebid-server/pull/2965/commits/28cadacc7d35635b256baf0dfe47260ed8c5584f)